new-log-viewer/src/components/Layout.css

```
@@ -0,0 +1,31 @@
+.container {
+    height: 100vh;
+}
+
+.menu-bar {
+    display: flex;
+    flex-direction: row;
+    height: 32px;
+    align-items: center;
+}
+
+.editor-container {
+    height: calc(100vh - 32px - 32px);
+}
+
+.status-bar {
+    align-items: center;
+    bottom: 0;
+    display: flex;
+    height: 30px;
```

```suggestion
    height: 32px;
```

We should probably declare some global variables in https://github.com/y-scope/yscope-log-viewer/blob/c939f7b55b55b42f65226470d5277b15ac484665/new-log-viewer/src/index.css#L11 and replace the references in `.menu-bar` and `.editor-container` to avoid inconsistencies.